JS/popupPage.js

```diff
@@ -0,0 +1,123 @@
+import { projectsarray, createProjects } from './projects.js';
+
+// popup page
+// get the div that includes the popup page
+
+const projectPopup = document.querySelector('.popupPage');
+const projectBtn = document.querySelectorAll('.projectBtn');
+
+projectBtn.forEach((btn, btnIndex) => {
+  btn.addEventListener('click', () => {
+    createProjects();
+    projectPopup.style.display = 'block';
+    const cardIndex = btnIndex;
+    projectPopup.innerHTML = `
+    <div class="popuppage-header">
+    <img
+      class="pageX"
+      src="assests/images/x.svg"
+      alt="close"
+    />
+    <img
+      src="${projectsarray[cardIndex].img}"
+      alt="Medical Illustration Sets"
+      class="poppage-desktop-img"
+    />
+  </div>
+  <div class="poppage">
+  <div class="desktop-content">
+      <div class="first-section">
+        <h1 class="title-desktop">
+        ${projectsarray[cardIndex].Name}
+        </h1>
+        <ul class="poppage-buttons">
```

Comment:
You have done pretty much fine creating the pop-up, however, there are a few concerns which are outlined below:

- The buttons are not correctly coloured, I recommend using the correct colour scheme for the button, and also they need a bit of alignment to the left as you can see in the Figma design
- Your pop-up doesn't have the transparent background overlay, that blurs out the page when the pop-up is fired

Please resolve the above mentioned issues to make the popup match the template

Your design
<img width="263" alt="effcwece" src="https://github.com/GeekyHacks/My-Personal-Portfolio/assets/84930355/bdf9b829-08ad-4ac3-b90b-867a9ce29152">

Figma design
<img width="200" alt="cascaca" src="https://github.com/GeekyHacks/My-Personal-Portfolio/assets/84930355/73ce128b-6cb4-4641-8535-002484715046">


Concerns for the popup
<img width="372" alt="overlay" src="https://github.com/GeekyHacks/My-Personal-Portfolio/assets/84930355/c8490884-21f6-486c-84c2-ae8ebe9c0d6e">